index.html 

```diff
@@ -0,0 +1,42 @@
+<! doctype html>
+<head>
+	<title> Bubble Tea site</title>
```

Comment:
Please use actual tabs throughout entire document.... these look like spaces.  Put this through an html prettifier online and then set up Visual Studio to do tab with two spaces.  Look this up online.

Reply:
Hi David, I have did changes to spacing. Can you confirm that you where able to pull the changes? Thank you.